server/task_executor_schema_update_ghost_sync.go

```diff
@@ -73,6 +73,11 @@ func getTableNameFromStatement(statement string) (string, error) {
 	return parser.GetExplicitTable(), nil
 }
 
+type ghostState struct {
```

Comment:
I prefer `sharedGhostState`.
Because it's **shared** state of gh-ost between tasks, not all of the gh-ost state.